src/shell/shell.cpp

```diff
@@ -944,8 +944,18 @@ static char const * const full_name="Z:\\COMMAND.COM";
 static char const * const init_line="/INIT AUTOEXEC.BAT";
 
 void SHELL_Init() {
-	/* Add messages */
-	MSG_Add("SHELL_ILLEGAL_PATH","Illegal Path.\n");
+	// Generic messages, to be used by any command or DOS program
+	MSG_Add("SHELL_ILLEGAL_PATH", "Illegal path.\n");
+	MSG_Add("SHELL_ILLEGAL_SWITCH", "Illegal switch: %s.\n");
+	MSG_Add("SHELL_MISSING_PARAMETER", "Required parameter missing.\n");
+	MSG_Add("SHELL_SYNTAX_ERROR", "Incorrect command syntax.\n");
+	MSG_Add("SHELL_ACCESS_DENIED", "Access denied - %s\n");
+	MSG_Add("SHELL_FILE_CREATE_ERROR", "File creation error - %s\n");
+	MSG_Add("SHELL_FILE_OPEN_ERROR", "File open error - %s\n");
+	MSG_Add("SHELL_FILE_NOT_FOUND", "File not found: %s\n");
+	MSG_Add("SHELL_FILE_EXISTS", "File %s already exists.\n");
```

Comment:
Consider enclosing filenames and strings with single quotes in the messages — I think it's nicer.

Reply:
OK, changed.